features/guidance/anticipate-lanes.feature

```diff
@@ -259,6 +259,140 @@ Feature: Turn Lane Guidance
             | waypoints | route                 | turns                                                   | lanes                                                                                                                                                  |
             | a,f       | abx,bcy,cdz,dew,ef,ef | depart,turn right,turn left,turn right,turn left,arrive | ,straight:false right:false right:true right:false,left:false left:true straight:false,straight:false right:true right:false,left:true straight:false, |
 
+    @anticipate
+        Scenario: Anticipate with lanes in roundabout: roundabouts as the unit of anticipation
+        Given the node map
+            |   |   | e |   |   |
+            | a | b |   | d | f |
+            |   |   | c |   |   |
+            |   |   |   |   |   |
+            |   |   | g |   |   |
+            | k | h |   | j | l |
+            |   |   | i |   |   |
+
+        And the ways
+            | nodes | turn:lanes:forward                      | highway | junction   | #   |
+            | ab    | slight_right\|slight_right&slight_right | primary |            |     |
```

Comment:
Remember to fix this once https://github.com/Project-OSRM/osrm-backend/pull/2635 is merged.